index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <p>LazyCoder</p>
+      <img src="./menu.png" alt="" />
+    </header>
+    <section>
+        <h1 id="title">
+          Hey There.<br />
+          I’m Mithlesh
+        </h1>
+        <p id="text1">I’am a Software Developer</p>
+        <p id="text2">
+          I can help you build a product , feature or website Look through some of
+          my work and experience! If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
+        <div id="social_menu">
+            <img src="./git.png" alt="">
+            <img src="./linkedin.png" alt="">
+            <img src="./angel.png" alt="">
+            <img src="./twitter.png" alt="">
+            <img src="./medium.png" alt="">
+        </div>
```

Comment:
The list of icons should use a `ul` tag to group them, and a `li` per icon; each icon should have a link (`a` tag)